stream-chat-android-client/src/main/java/io/getstream/chat/android/client/ChatClient.kt

```diff
@@ -927,5 +935,8 @@ public class ChatClient internal constructor(
 
         public val isInitialized: Boolean
             get() = instance != null
+
+        public fun isValid(remoteMessage: RemoteMessage, defaultNotificationConfig: NotificationConfig = NotificationConfig()): Boolean =
```

Comment:
NIT: I think it might be worth it to give all `isValid` methods  (except RemoteMessage extension function)  a more descriptive name, e.g. `isValidRemoteMessage`. What do you think?